prelude/align.md

```diff
@@ -0,0 +1,47 @@
+This is basically a copy of the `Align` type in the Rust compiler.
+See [Align](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_target/abi/struct.Align.html).
+
+`Align` is always a power of two.
+
+```rust
+pub struct Align { pow2: BigInt }
+
+impl Align {
+    pub const ONE: Align = Align { pow2: BigInt::from(0) };
+
+    pub fn from_bits(bits: impl Into<BigInt>) -> std::result::Result<Align, String> {
+        let bytes = Size::from_bits(bits).bytes();
+        Align::from_bytes(bytes)
+    }
+
+    pub fn from_bytes(align: impl Into<BigInt>) -> std::result::Result<Align, String> {
```

Comment:
Might be nicer to make this infallible by always using the next power of two. (`BigInt` should have a `next_power_of_two` method for this.)